packages/core/src/transport/httpRequest.ts

```diff
@@ -1,5 +1,5 @@
 import { EndpointBuilder } from '../domain/configuration/endpointBuilder'
-import { monitor, addErrorToMonitoringBatch, addMonitoringMessage } from '../domain/internalMonitoring'
+import { monitor, addErrorToMonitoring, addMonitoringMessage } from '../domain/internalMonitoring'
```

Comment:
It would be more natural to rename `addErrorToMonitoring` to `addMonitoringError`, to follow the `addMonitoringMessage` name.

Reply:
Yes, could be nice